cmd/create/oidcconfig/cmd.go

```diff
@@ -207,14 +214,14 @@ func run(cmd *cobra.Command, argv []string) {
 
 	if !args.managed {
 		if !args.rawFiles {
-			if r.Reporter.IsTerminal() {
```

Comment:
I see in some places in the code we have `!output.HasFlag() && r.Reporter.IsTerminal()` and in other places we have `!output.HasFlag() || r.Reporter.IsTerminal()`.

So in case we need json/yaml output for automation we don't want to print the info messages, right?